test/acceptance/_helpers.bash

```diff
@@ -128,3 +128,32 @@ wait_for_ready() {
     echo "${POD_NAME} never became ready."
     exit 1
 }
+
+wait_for_complete_job() {
+	POD_NAME=$1
+
+    check() {
+        # This requests the pod and checks whether the status is running
+        # and the ready state is true. If so, it outputs the name. Otherwise
+        # it outputs empty. Therefore, to check for success, check for nonzero
+        # string length.
+        kubectl get job $1 -o json | \
+            jq -r 'select(
+                .status.succeeded == 1 
+            ) | .metadata.namespace + "/" + .metadata.name'
+    }
+
+    for i in $(seq 60); do
+        if [ -n "$(check ${POD_NAME})" ]; then
+            echo "${POD_NAME} is complete."
+            sleep 10
```

Comment:
Not a blocking comment, but this test could run for a long time. I'm wondering if 3-5s is more appropriate?

Reply:
Lowered!